README.md

```diff
@@ -85,6 +85,27 @@ digraph foo {
 ....
 ```
 
+## Using Your Own Kroki
+
+By default this extension sends information and receives diagrams back from `https://kroki.io`.
+
+For significant usage or many users we recommend setting up your own server (and also if sending information to an external server is considered an information security risk).
```

Comment:
I think "For significant usage or many users we recommend setting up your own server" is misleading because it gives the impression that the public instance is somehow unstable and unreliable. The public instance can handle significant usage and many users. The infrastructure has been designed with this goal in mind.

In my opinion, the main reasons to use your own server are:
- Network restrictions (ie. Kroki is not available behind your corporate firewall)
- Network latency (ie. you are far from the European public instance)
- Privacy (ie. you don't want to send your diagrams to a remote server on the Internet)

Could you please rephrase in this way?

Reply:
Thanks, I was struggling with how to get the nuance right here.